src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/mapView.js

```diff
@@ -23,34 +23,40 @@ styleHead.innerText = `.mapboxgl-map {
         }`
 document.head.appendChild(styleHead);
 
+const TOKEN = 'pk.eyJ1IjoiZW5zby1vcmciLCJhIjoiY2tmNnh5MXh2MGlyOTJ5cWdubnFxbXo4ZSJ9.3KdAcCiiXJcSM18nwk09-Q';
+
+const GEO_POINT         = "GeoPoint";
+const GEO_MAP           = "GeoMap";
+const SCATTERPLOT_LAYER = "ScatterplotLayer";
+const DEFAULT_RADIUS    = 150;
```

Comment:
I asked for constants to be in appropriate section on top of the file. They are neither on top, nor in a section.

Reply:
Plus, pls, add docs to the file, as they are missing now.